package.json

```diff
@@ -18,7 +18,7 @@
         "build": "vite build",
         "dev": "vite",
         "lint": "eslint src --ext js,cjs,mjs,jsx,ts,tsx",
-        "netlify": "ntl dev",
```

Comment:
@kevwhuang I wasn't sure why ntl was the command here. Are you using https://github.com/ruyadorno/ntl ?

Reply:
I think it's a command from the Netlify CLi and "ntl" is a command shortcut. However, I don't think we're gonna need to use the Netlify dev server so I'm going to drop it when I go back and change it to Vite.